packages/strapi-admin/controllers/Admin.js

```diff
@@ -94,5 +94,43 @@ module.exports = {
       strapi.reload.isWatching = true;
       ctx.badRequest(null, [{ messages: [{ id: 'An error occurred' }] }]);
     }
+  },
+
+  /**
+   * Create a/an admin record.
+   *
+   * @return {Object}
+   */
+
+  create: async (ctx) => {
+    const values = ctx.request.body;
+
+    if (values.password) {
+      values.password = await strapi.plugins['users-permissions'].services.user.hashPassword(values);
+    }
+
+    const data = await strapi.query('admin', 'users-permissions').create(values);
+
+    // Send 201 `created`
+    ctx.created(data);
+  },
+
+  /**
+   * Update a/an admin record.
+   *
+   * @return {Object}
+   */
+
+  update: async (ctx) => {
+    const values = ctx.request.body;
+
+    if (values.password) {
+      values.password = await strapi.plugins['users-permissions'].services.user.hashPassword(values);
+    }
+
+    const data = await strapi.query('admin', 'users-permissions').update(_.assign(ctx.params, values));
```

Comment:
I don't know if the templates are similar but we should use Object.assign() instead of _.assign.